configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
+# QUOTA
+- name: ratelimit_by_service_by_consumer
+  quota_params:
```

Comment:
I think the idea of hoisting the template types and template constructors above from the params is a good idea.
I have added a issue to address this https://github.com/istio/mixer/issues/803